src/main/java/org/omnifaces/util/Ajax.java

```diff
@@ -312,7 +312,7 @@ public static void load(String libraryName, String resourceName) {
             throw new IllegalArgumentException(ERROR_NO_SCRIPT_RESOURCE);
         }
 
-        try (Scanner scanner = new Scanner(resource.getInputStream(), UTF_8.name())) {
+        try (Scanner scanner = new Scanner(resource.getInputStream(), UTF_8)) {
```

Comment:
Great find. Hint for future PRs: it's a much better idea if you create as much isolated as possible PR whereby you simply replace all occurrences of UTF_8.name() by UTF_8 without touching rest of the code.

Reply:
the main bad idea was the initial fork without PR back ... 
I've diverged too much and I tried to realign